server/routes/v1/upload.ts

```diff
@@ -68,6 +68,13 @@ export const postUpload = [
         })
       }
 
+      if (mode !== 'newVersion' && mode !== 'newModel') {
```

Comment:
I appreciate I made this modes originally, but I wonder if we can make them into a type as we reuse them in various places